packages/ui/src/composites/wui-input/styles.ts

```diff
@@ -0,0 +1,72 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    position: relative;
+  }
+
+  input {
+    width: 100%;
+    border-radius: var(--wui-border-radius-xxs);
+    border: 1px solid var(--wui-overlay-005);
+    background: var(--wui-overlay-005);
+    font-size: var(--wui-font-size-paragraph);
+    font-weight: var(--wui-font-weight-regular);
+    letter-spacing: var(--wui-letter-spacing-paragraph);
+    color: var(--wui-color-fg-100);
+    transition: all 200ms ease-in-out 35ms;
```

Comment:
Can add `carret-color` property to match current accent color for 🔥 points 😄 
Similarly we could also set global `:selection` background color to be accent and text color to be inverse of that.

Reply:
Good one! Will set `caret-color` for now. Let's add `:selection` later on.